app/components/WardTile.tsx

```diff
@@ -0,0 +1,61 @@
+import { tw } from '@helpwave/common/twind/index'
+import type { Languages } from '@helpwave/common/hooks/useLanguage'
+import type { PropsWithLanguage } from '@helpwave/common/hooks/useTranslation'
+import { useTranslation } from '@helpwave/common/hooks/useTranslation'
+import type { FunctionComponent } from 'react'
+import Bed from '../icons/Bed'
+import { PillLabelBox } from './PillLabelBox'
+
+type WardTileTranslation = {
+  edit: string
+}
+
+const defaultWardTileTranslations: Record<Languages, WardTileTranslation> = {
+  en: {
+    edit: 'Edit'
+  },
+  de: {
+    edit: 'Bearbeiten'
+  }
+}
+
+export type WardTileProps = {
+  wardName: string,
+  roomNames: string[],
+  bedCount: number,
+  unscheduledTasks: number,
+  inProgressTasks: number,
+  doneTasks: number,
+  isSelected: boolean,
+  tileClickedCallback: () => void,
```

Comment:
Default values for callbacks? Like `() => {}`?

Reply:
You cannot set a default value on a type definition as far as I know. Do you want me to a default Object like
```
const default = {
    tileClickedCallback: () => {}
}
```

Reply:
Take a look at [this example](https://github.com/helpwave/web/blob/main/app/components/ProfilePicture.tsx#L9) for default values for props.